src/pages/students/StudentsOverview.jsx

```diff
@@ -1,7 +1,141 @@
+import Sidebar from '../../components/SideBar';
+import React, { useEffect, useState } from 'react';
+import '../../styles/Students.css';
+import Table from '@mui/material/Table';
+import TableBody from '@mui/material/TableBody';
+import TableCell from '@mui/material/TableCell';
+import TableContainer from '@mui/material/TableContainer';
+import TableHead from '@mui/material/TableHead';
+import TableRow from '@mui/material/TableRow';
+import Paper from '@mui/material/Paper';
+import TablePagination from '@mui/material/TablePagination';
+import axios from 'axios';
+import BootstrapButton from '../../components/btnBlue';
+import BootstrapButtonRed from '../../components/BtnRed';
+
+const baseURL = 'http://localhost:8000/student';
+const columns = [
+  { id: 'id', label: 'ID', align: 'left' },
+  { id: 'code', label: 'Code', align: 'right' },
+  { id: 'name', label: 'Name', align: 'right' },
+  { id: 'email', label: 'Email', align: 'right' },
+  { id: 'phone', label: 'Phone', align: 'right' },
+  { id: 'status', label: 'Status', align: 'right' },
+  { id: 'createdAt', label: 'CreatedAt', align: 'right' },
+  { id: 'updatedAt', label: 'UpdatedAt', align: 'right' }
+];
+
 const StudentsOverview = () => {
+  const [page, setPage] = useState(0);
+  const [rowsPerPage, setRowsPerPage] = useState(5);
+  const [data, setData] = useState([]);
+  const [info, setInfo] = useState({});
+
+  const token = localStorage.getItem('token');
+  useEffect(() => {
+    axios
```

Comment:
Take this request out the component

Reply:
Listo

Reply:
listo he sacado la solicitud del componente